gwcli/storage.py

```diff
@@ -207,14 +208,20 @@ def ui_command_create(self, pool=None, image=None, size=None, count=1):
             if not str(count).isdigit():
                 self.logger.error("invalid count format, must be an integer")
                 return
+        if ring_buffer_size:
+            if not str(ring_buffer_size).isdigit():
```

Comment:
Should there be a check for upper bound for buffer size? I think the kernel caps it anyway, correct?